frontend/src/lib-components/molecules/ExpandingInfo.tsx

```diff
@@ -71,6 +71,8 @@ export default function ExpandingInfo({
   const { colors } = useTheme()
   const [expanded, setExpanded] = useState<boolean>(false)
 
+  if (info === null) return <div>{children}</div>
```

Comment:
I don't think this kind of optional rendering belongs in this component. Conceptually I don't think it makes any sense so use this component when `info` is `null`. I think conditional rendering should be at the using end.

Reply:
The conditional rendering of such wrapper component caused tons of annoying boilerplate. I agree this way feels a bit dirty too though.. Should I build some higher order component to do the conditional wrapping?

Reply:
Yeah I would create a new component somewhere around the vasu code that would conditionally wrap other components in the `ExpandingInfo` component.